internal/gh/ghui/push.go

```diff
@@ -543,26 +543,15 @@ func createPRMetadata(branch meta.Branch, vm *GitHubPushModel) actions.PRMetadat
 }
 
 // Compare local metadata with PR metadata for any changes
+// If something error occurs, return true to be safe
 func isDifferencePRMetadata(avbr meta.Branch, vm *GitHubPushModel) bool {
 	local := createPRMetadata(avbr, vm)
 
-	prs, err := vm.getPRs()
+	pr, err := vm.client.PullRequest(context.Background(), avbr.PullRequest.ID)
```

Comment:
The direction looks good. Can you change the logic slightly so that we don't have to make the API call twice (once in this and another in getPRs)?

Reply:
Sorry for the delayed response. I updated